<ui version="4.0" >
 <class>TabExternalToolsWidget</class>
 <widget class="QWidget" name="TabExternalToolsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>External Tools</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="Q3GroupBox" name="psToolGroupBox" >
     <property name="title" >
      <string>PostScript Interpreter</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="psToolLabel" >
          <property name="text" >
           <string>&amp;Name of Executable:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>psToolLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="psToolLineEdit" >
          <property name="toolTip" >
           <string>&lt;qt>Add the path for the Ghostscript interpreter. On Windows, please note it is important to note you need to use the program named gswin32c.exe - NOT gswin32.exe. Otherwise, this maybe cause a hang when starting Scribus.&lt;/qt></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="psToolChangeButton" >
          <property name="text" >
           <string>&amp;Change..</string>
          </property>
          <property name="shortcut" >
           <string>Alt+C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="psAntialiasTextCheckBox" >
          <property name="toolTip" >
           <string>Antialias text for EPS and PDF onscreen rendering</string>
          </property>
          <property name="text" >
           <string>Antialias &amp;Text</string>
          </property>
          <property name="shortcut" >
           <string>Alt+T</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="psAntialiasGraphicsCheckBox" >
          <property name="toolTip" >
           <string>Antialias graphics for EPS and PDF onscreen rendering</string>
          </property>
          <property name="text" >
           <string>Antialias &amp;Graphics</string>
          </property>
          <property name="shortcut" >
           <string>Alt+G</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="psResolutionLabel" >
          <property name="text" >
           <string>Resolution:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>psResolutionSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="psResolutionSpinBox" >
          <property name="suffix" >
           <string> dpi</string>
          </property>
          <property name="maximum" >
           <number>2400</number>
          </property>
          <property name="minimum" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="imageToolGroupBox" >
     <property name="title" >
      <string>Image Processing Tool</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="imageToolLabel" >
        <property name="text" >
         <string>Name of &amp;Executable:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>imageToolLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imageToolLineEdit" >
        <property name="toolTip" >
         <string>&lt;qt>File system location for graphics editor. If you use gimp and your distribution includes it, we recommend 'gimp-remote', as it allows you to edit the image in an already running instance of gimp.&lt;/qt></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imageToolChangeButton" >
        <property name="text" >
         <string>&amp;Change...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="extBrowserToolGroupBox" >
     <property name="toolTip" >
      <string>Web Browser to launch with links from the Help system</string>
     </property>
     <property name="title" >
      <string>Web Browser</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="extBrowserToolLabel" >
        <property name="text" >
         <string>Name of &amp;Executable:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>imageToolLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="extBrowserToolLineEdit" >
        <property name="toolTip" >
         <string>&lt;qt>File system location for your web browser. This is used for external links from the Help system.&lt;/qt></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="extBrowserToolChangeButton" >
        <property name="text" >
         <string>&amp;Change...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>161</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="rescanButton" >
       <property name="toolTip" >
        <string>Rescan for the external tools if they do not exist in the already specified location</string>
       </property>
       <property name="text" >
        <string>&amp;Rescan</string>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
